lib/rucio/common/utils.py

```diff
@@ -1896,3 +1896,28 @@ def integer(self):
         delete_value = (self.delete or self.delete is None) * 1
 
         return read_value + write_value + delete_value
+
+
+def retrying(retry_on_exception, wait_fixed, stop_max_attempt_number):
```

Comment:
I know this is already a bit older, so sorry for that. But please add type annotations here 😄

Reply:
:trollface:  But it passed all the tests with a green check mark, meaning it's good, isn't it ? :trollface: :grin:

Reply:
:trollface: